lib/bokken_web/controllers/team_controller.ex

```diff
@@ -3,6 +3,7 @@ defmodule BokkenWeb.TeamController do
 
   alias Bokken.Events
   alias Bokken.Events.Team
+  import Bokken.Guards
```

Comment:
Maybe we could move this import to `lib/bokken_web.ex` to `controller`. What do you guys think?

Reply:
Right, I'm going to implement it 👍